### DISCUSSION TITLE
TerriaJS Version 8 documentation

### DISCUSSION THREAD

**Discussion by tephenavies** [TerriaJS/terriajs#5099] [General]
We now have published documentation for TerriaJS v8! It's hosted at https://docs-v8.terria.io/ and includes documentation generated from our new Traits system for Terria supported data types and some documentation that existed for TerriaJS version 7 that we've tried to update or mark as outdated.

**Comment by DavidHine** [comment]
Hi @steve9164 and TerriaJS community
Best wishes for a happy 2021 where many factors bring  much needed relief from uncertainty and worry about the pandemic and its consequences.
As @ushermawan mentioned, I would also welcome a clear guide to regionMapping. If we get the bare bones of the steps, I am happy to flesh out the detail for developers wanting to implement regionMapping.
I follow the purpose and the requirements in general but linking the polygons to the csv files has not worked for me. I must be getting the details of the shared ID wrong. I also saw that steve commented that the explanation on the Getting Started pages was now out of date. Although there are examples, it is really helpful to have the critical parts explained as well as demonstrated.
